docs/content/latest/deploy/checklist.md

```diff
@@ -73,12 +73,13 @@ YugabyteDB explicitly manages a block cache, and doesn't need the entire data se
 
 - SSDs (solid state disks) are required. 
 - Both local or remote attached storage work with YugabyteDB. Since YugabyteDB internally replicates data for fault tolerance, remote attached storage which does its own additional replication is not a requirement. Local disks often offer better performance at a lower cost.
-- Multi-disk nodes
-      - Do not use RAID across multiple disks. YugabyteDB can natively handle multi-disk nodes (JBOD).
+- Multi-disk nodes  
+      - Do not use RAID across multiple disks. YugabyteDB can natively handle multi-disk nodes (JBOD).  
```

Comment:
ditto

Reply:
The spaces are converted to new lines and is used for child unordered list.
See 1st picture, there are no child lists

Reply:
Makes sense ? If yes I'll merge this.
@stevebang